<launch>
    <arg name="rviz" default="true"/>
    <arg name="location" default="sargans"/>
    <node pkg="tf2_ros" exec="static_transform_publisher" name="world_map" args="--frame-id world --child-frame-id map"/>

    <node pkg="grid_map_geo" exec="test_tif_loader" name="test_tif_loader" namespace="grid_map_geo" output="screen">
        <param name="tif_path" value="/vsizip/vsicurl/https://terrain.ardupilot.org/SRTM1/N09E047.hgt.zip/N09E047.hgt"/>
        <param name="tif_color_path" value=""/>
    </node>

    <group if="$(var rviz)">
        <node pkg="rviz2" exec="rviz2" name="rviz2" args="-d $(find-pkg-share grid_map_geo)/rviz/config.rviz"/>
    </group>
</launch>
